### DISCUSSION TITLE
v2.0

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Avvie#18] [Announcements]
I'm excited to bring you this major update to Avvie which features a full port to GTK4+Libadwaita. With this it brings:
 - Much improved performance.
 - The new GNOME 42 Adwaita style.

There no major new features, but there are the following new changes:
 - Crop can now be locked to a user given aspect ratio.
 - There is now a circle guide in the crop selector.

As a bonus there is also a new pink theme! :cherry_blossom: Which is the new default. You can change it in app preferences. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Taiko2k/Avvie/releases/tag/'>v2.0</a>.</em>

**Comment by Solarunit** [comment]
Thanks for the update!
Just wondering, I have accidentally removed 2 out of 3 circle previews at the bottom by middle clicking on them. And now I have no idea how to bring them back  :)

**Reply by Taiko2k** [reply]
There's a button to add them in the preferences window.

**Reply by Solarunit** [reply]
Now I see, ty.

**Comment by Solarunit** [comment]
Some other things I have noticed:

- Open image file button icon is not very obvious: 
![](https://i.imgur.com/H6xfpgZ.png)

(probably depends on the system icons, Papirus-Dark here)

- App icon of the app is just a "placeholder" icon on Budgie: 
![](https://i.imgur.com/1xf5Zq9.png)


- The Crop toggle is too big imo:

![](https://i.imgur.com/uH44ou3.png)


_____

BTW you app was reviewed: https://www.youtube.com/watch?v=038Jz6naLPA
